evap/student/templates/student_vote.html

```diff
@@ -43,6 +40,35 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
     <form id="student-vote-form" method="POST" class="form-horizontal">
         {% csrf_token %}
 
+        {% if participants_warning %}
+            <div class="card card-outline-warning mb-3">
+                <div class="card-header">
+                    <span class="fas fa-exclamation-triangle"></span> {% trans 'Small number of participants' %}
+                </div>
+                <div class="card-body">
+                    {% blocktrans %}
+                        Only a small number of people can take part in this evaluation. You should be aware that
+                        contributors might be able to guess who voted for a specific answer and who wrote a text
+                        answer. Results will only be published, if two or more people participated in the evaluation.
+                    {% endblocktrans %}
+                    {% if course.num_voters == 0 %}
+                        <br />
+                        {% blocktrans %}
+                            If you want your text answers to be shown to the contributors even if you remain the only
+                            person taking part in this evaluation, please check the box below.
+                        {% endblocktrans %}
+
+                        <div class="form-check mt-3">
+                            <input class="form-check-input" id="text_results_publish_confirmation" type="checkbox" name="text_results_publish_confirmation" />
+                            <label class="form-check-label" for="text_results_publish_confirmation">
+                                {% trans 'Show text answers to the contributors even if I remain the only person who takes part in this evaluation.' %}
+                            </label>
+                        </div>
```

Comment:
so, you made this opt-in. was that a conscious decision?

alternative opt-out suggestion:

```
Generally, results will only be published if two or more people participated in the evaluation.
Text answers, however, will be shown to contributors even if only one person participated.

You're the first person participating in this evaluation. If you're worried about your anonymity, use this:
[ ] Please do not show my comments to contributors if I remain the only person who takes part in this evaluation.
```

in case it remains opt-in, please think about adding a second disclaimer "Your comments won't be shown to anyone in case you remain the only voter" or even a second checkbox at the bottom. Because what if someone initially does not care, does not check the checkbox, then writes actually valuable comments, but forgets about the checkbox?

Reply:
i did consider opt-out but went for opt-in following standard data privacy regulations where opt-out is never allowed.
an additional information with checkbox on the bottom (that is removed once the upper checkbox was selected) sounds good.